src/psyclone/psyGen.py

```diff
@@ -1809,9 +1809,20 @@ def __init__(self, KernFactory, BuiltInFactory, alg_calls=None):
         self._opencl = False  # Whether or not to generate OpenCL
         # InvokeSchedule opencl_options default values
         self._opencl_options = {"end_barrier": True}
+        # TODO: #312 Currently NameSpaceManager and SymbolTable coexist, but
+        # it would be better to merge them
```

Comment:
Please also mention that the SymbolTable is currently only used to manage the import of global symbols that were previously inside kernels.

Reply:
Extended comment as suggested